test/sanity/code-smell/package-data.py

```diff
@@ -61,6 +61,7 @@ def assemble_files_to_ship(complete_file_list):
         'hacking/test-module',
         'hacking/test-module.py',
         '.cherry_picker.toml',
+        '.flake8',
```

Comment:
You may need to include this into https://github.com/ansible/ansible/blob/devel/MANIFEST.in

Reply:
I don't really think so, for the same reason about why we don't include `.cherry_picker.toml`. Summarizing, it's a dev only thing which wouldn't really need to make its way to any installation nor package.